spec/system/support_interface/docs_spec.rb

```diff
@@ -18,7 +18,10 @@ def given_i_am_a_support_user
   end
 
   def when_i_visit_the_process_documentation
+    allow(ApplicationStateChange).to receive(:workflow_spec).and_return(Struct.new(:states).new([]))
+
     visit support_interface_docs_provider_flow_path
+    expect(ApplicationStateChange).to have_received(:workflow_spec).exactly(3).times
```

Comment:
Maybe we can move this to its own `then` step, as I think usually we should avoid any expectations in `when` steps, what do you think?

Reply:
Good point. I have moved the expectations to their own `then` steps.